swan-lake/development-tutorials/ballerina-central/publish-packages-to-ballerina-central.md

```diff
@@ -126,40 +125,36 @@ $ bal push
 ```
 
 ### Publish a new version of a package
-If you require adding new features/improvements/fixes to a library package which you have already published to Ballerina central, 
-you are allowed to publish them under a new version, based on the [Semantic Versioning Specification](https://semver.org/).
-However, it's the library developer's responsibility to be cautious when deciding on the new package versions
-(especially when there are potential breaking/backward-incompatible API changes), as otherwise, it may result in library
-versions that are compatible only by the version but not by the implementation.
+If you require adding new features/improvements/fixes to a package, which you have already published to Ballerina central, you are allowed to publish them under a new version, based on the [Semantic Versioning Specification](https://semver.org/).
+
+However, it's the package developer's responsibility to be cautious when deciding on the new package versions (especially when there are potential breaking/backward-incompatible API changes), as otherwise, it may result in package versions that are compatible only by the version but not by the implementation.
 
->**Tip:** As a precaution, use the Ballerina semver validator CLI tool (experimental) to check if your new API changes
-conform to the version that you are trying to publish to Ballerina central.
+>**Tip:** As a precaution, use the Ballerina semver validator CLI tool (experimental) to check if your new API changes conform to the version that you are trying to publish to Ballerina central.
 
-By default, running the `bal semver` command on the root directory of the package will compare the local changes with the 
-"closest compatible" published version available in Ballerina Central. 
-(Use `bal semver --help` for the CLI help text which outlines all the available command options)
+By default, running the `bal semver` command on the root directory of the package will compare the local changes with the "closest compatible" published version available in Ballerina Central. (Use `bal semver --help` for the CLI help text which outlines all the available command options)
 
 > **Note:** Semver validator CLI support is only available from Swan Lake Update 2 onwards.
 
 ## Use the packages in Ballerina Central
 
-After publishing your first package, you can create a second package and use the already published package in it.
-Any package published in Ballerina Central is public and can be used in other packages.
- For more information, see [Import a module](/learn/manage-dependencies/#import-a-module).
+After publishing your first package, you can create a second package and use the already published package in it. Any package published in Ballerina Central is public and can be used in other packages.
```

Comment:
```suggestion
After publishing your first package, you can create a second package and use the already published package in it.
```
Removing this part since we have private packages as well.